### DISCUSSION TITLE
How can I remove message in console ?

### DISCUSSION THREAD

**Discussion by bega-ekta** [WalletConnect/walletconnect-monorepo#2335] [Developer Support] [answered]
I am getting this message in the console -> "Unsuccessful attempt at preloading some images". 
How can I remove that? Thanks.

![Screenshot_69](https://github.com/WalletConnect/walletconnect-monorepo/assets/115062160/71cc418f-77a5-4940-a143-8e5efe18e742)

**Comment by finessevanes** [comment]
Could you please share which SDK you're using and provide us with more information on your code implementation?

**Reply by bega-ekta** [reply]
I am using the wallet connect versions below.
The project is a react-ts project created with vite@4.2.0.
I am also using wagmi

![Screenshot_78](https://github.com/WalletConnect/walletconnect-monorepo/assets/115062160/ff490845-f6d1-47a3-9c62-8cde0949413d)
![Screenshot_76](https://github.com/WalletConnect/walletconnect-monorepo/assets/115062160/9409be5d-a5d2-4036-bb51-da89e62fb47c)

**Comment by glitch-txs** [comment]
Seems like some issue with the project ID ig

Are you setting it correctly?

**Reply by bega-ekta** [reply]
Yes. Project ID from Wallet Connect Dashboard

**Comment by finessevanes** [comment]
Hey - I created a vite dapp using `npm create vite@latest` and selected Typescript.

I removed everything in `App.tsx` and replaced it with this:
```ts
import { EthereumClient, w3mConnectors, w3mProvider } from '@web3modal/ethereum'
import { Web3Modal, Web3Button } from '@web3modal/react'
import { configureChains, createConfig, WagmiConfig } from 'wagmi'
import { arbitrum, mainnet, polygon } from 'wagmi/chains'

const chains = [arbitrum, mainnet, polygon]
const projectId = '.....'

const { publicClient } = configureChains(chains, [w3mProvider({ projectId })])
const wagmiConfig = createConfig({
  autoConnect: true,
  connectors: w3mConnectors({ projectId, version: 1, chains }),
  publicClient
})
const ethereumClient = new EthereumClient(wagmiConfig, chains)

function App() {
  return (
    <>
      <WagmiConfig config={wagmiConfig}>
        <h1>Hello</h1>
        <Web3Button />
      </WagmiConfig>

      <Web3Modal projectId={projectId} ethereumClient={ethereumClient} />
    </>
  )
}

export default App;
```

Here's my package.json:
```json
  "dependencies": {
    "@web3modal/ethereum": "^2.4.1",
    "@web3modal/react": "^2.4.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "viem": "^0.3.24",
    "wagmi": "^1.0.4"
  },
  "devDependencies": {
    "@types/react": "^18.0.28",
    "@types/react-dom": "^18.0.11",
    "@typescript-eslint/eslint-plugin": "^5.57.1",
    "@typescript-eslint/parser": "^5.57.1",
    "@vitejs/plugin-react": "^4.0.0",
    "eslint": "^8.38.0",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.3.4",
    "typescript": "^5.0.2",
    "vite": "^4.3.2"
  }
```

Not seeing any logs like you have above:
<img width="1510" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/44731696/d241ad6a-cd37-41ee-8aa5-04893f11b507">


I would suggest updating to the latest `web3modal/ethereum`, `web3modal/react`, and `wagmi` packages.

**Reply by bega-ekta** [reply]
Thanks. I will update my packages.

**Reply by ikaz-jem** [reply]
i have the same issue

**Reply by ikaz-jem** [reply]
updating things there is not working i

**Comment by ikaz-jem** [comment, marked as answer]
i found the real answer i had the same issue , im using react npx , typescript 
problem is not in your app its on wallet connect link to fetch bsc testnet icon , remove BSC testnet from your chain List and the msg will disappear,
if still having same problem , take a look on your switch-chain component , the chain that appears without real icon just remove it from chain list 
i know that the fetching error is fatal and expose your projectID as well just try this out  100% will resolve your issue good luck

**Reply by ikaz-jem** [reply]
also do this if you want to leave BSC testnet : 
 chainImages= {
{  97: "https://explorer-api.walletconnect.com/v3/logo/lg/0e11ef1b-ba1a-44df-4de2-9f5225677c00?projectId=''replace with your projectID",

pass thees props to modal component

**Reply by finessevanes** [reply]
Thank you for getting to the bottom of it.
